### DISCUSSION TITLE
Problem with height and width of cropped image

### DISCUSSION THREAD

**Discussion by dawidgola** [advanced-cropper/vue-advanced-cropper#206] [Q&A]
Why sample photo in size (275 × 183 px) on cropper is so much and when we do crop it doesn't crop on my scencil that is width: 834, height: 224 
I would like to get the result that when I upload a small or huge photo it will be cropped to the dimensions width: 834, height: 224 
example : https://codesandbox.io/s/vue-advanced-cropper-options-api-forked-bqmqgo?file=/src/App.vue:152-176

**Comment by Norserium** [comment]
@dawidgola, hello.

I should to clarify, what you want to get.

It's pretty unlikely that you want to retrieve the part of image that has width equals to 834 pixels and height equals to 224 pixels. Cause sometimes it's impossible at all if an image smaller than this size and it's pointless if an image noticeable larger than this size.

I think that you want something else: you want to preserve specific aspect ratio and resize the result to the specific dimension no matter what image part was cropped.

Let's see what did you use to achieve it:
- `stencil-size` prop sets the size of stencil in pixels, not the cropped area (result), just the element that represents the stencil area;
- `default-size` prop sets the default size of the cropped area (result), if it's possible to set this size
- `image-restriction` should have one of this values `"fit-area"`,  `"fill-area"`,  `"stencil"`,  `"none"`

Long story short, it won't help you to achieve you what you want to get.

And I suppose you want to get [this](https://codesandbox.io/s/vue-advanced-cropper-options-api-fixed-h2xhwv?file=/src/App.vue).
